eng/pipelines/templates/variables/globals.yml

```diff
@@ -4,10 +4,7 @@ variables:
   AdditionalTestArguments: ''
   skipComponentGovernanceDetection: true
   nugetMultiFeedWarnLevel: 'none'
-  ${{ if ne(variables['System.TeamProject'], 'internal') }}:
-    BuildConfiguration: 'Debug'
-  ${{ if eq(variables['System.TeamProject'], 'internal') }}:
-    BuildConfiguration: 'Release'
+  BuildConfiguration: 'Release'
```

Comment:
I would not do this in all cases as it might make debugging tests more difficult.

Reply:
If we want to only ensure pack is release we should hardcode the build configuration for the pack command directly.

Reply:
I made this change @weshaggard